codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoderTest.java

```diff
@@ -200,20 +201,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
     }
 
     @Test
-    public void dataWriteAfterGoAwayShouldFail() throws Exception {
```

Comment:
Are we testing this elsewhere?  Perhaps we should test that we can still write after go away (under the correct conditions) for each frame type?